src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -224,22 +229,21 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
     const results = [
       {
         title: 'Export',
+        disabled: false,
         onClick: () => {
           exportManifest(uuid, name);
         }
-      }
-    ];
-    if (user.canWriteManifests) {
-      results.push({
+      },
+      {
+        disabled: true,
```

Comment:
This should be disabled based on the permissions of the user, not permanently disabled
```suggestion
        disabled: !user.canWriteManifests,
```